task/docker-build/0.1/docker-build.yaml

```diff
@@ -20,6 +20,9 @@ spec:
   - name: builder_image
     description: The location of the docker builder image.
     default: docker.io/library/docker:stable@sha256:18ff92d3d31725b53fa6633d60bed323effb6d5d4588be7b547078d384e0d4bf #tag: stable
+  - name: dind_image
+    description: The location of the docker-in-docker image.
+    default: docker:dind
```

Comment:
can you please document this in README as well?

Reply:
Shouldn't be it `docker:19-dind` too ?

Reply:
the tests are working fine so I guess we can stick to `docker:dind`. WDYT?

Reply:
I mean the nightly is green. ref:- https://prow.tekton.dev/?type=periodic&job=periodic-tekton-catalog-integration-tests

Reply:
I've updated the README.

I don't think the default value of `dind_image` should be `docker:19-dind`. Rolling back to `19-dind` is a temporary fix for circumventing issues with the latest stable docker release. In the long run, `docker:dind` is a more appropriate default value.